roles/service-nginx-git-homepage/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: install packages
+  package:
+    name: "git"
+    state: present
+
+- name: get repos
+  git:
+    repo: "{{ item.git_repo }}"
+    dest: "{{ item.path }}"
+  loop: "{{ git_homepages|flatten(levels=1) }}"
```

Comment:
Ist das 'flatten(levels=1)' erforderlich? Kann man das nicht einfach weglassen?

Reply:
Nein, ist es in der Tat nicht...